cves/CVE-2020-7471.yml

```diff
@@ -16,7 +16,7 @@ curated_instructions: |
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
 curated: false
-reported_instructions: |
+reported_instructions: true
```

Comment:
I think you accidentally changed this to true.

Reply:
Ye, my bad